### DISCUSSION TITLE
Make "On The Air" - Listeners, x Unique Clickable

### DISCUSSION THREAD

**Discussion by Countryman98** [AzuraCast/AzuraCast#5952] [Ideas]
Hi Guys,

I think a way of making the "On The Air Dashboard" - "Listeners, 23 Unique" Clickable directly to https://myjoint.live/station/1/reports/listeners rather than having to go the long way through "Reports -> Listeners
![Capture](https://user-images.githubusercontent.com/65417549/208311014-dc4534ec-9053-4594-a2ee-208696c53c5e.JPG)

**Comment by BusterNeece** [comment]
@Countryman98 If you click the listener count on the main dashboard, it does exactly that. Welcome to submit a feature request to add it there too: https://features.azuracast.com/
